Dockerfile

```diff
@@ -0,0 +1,15 @@
+# ---- Base image - order important ----
+FROM hmcts/ccd-definition-processor:latest as base
```

Comment:
Shouldn't we use this repo:
https://github.com/hmcts/div-ccd-definitions/pull/159/files#diff-3254677a7917c6c01f55212f86c57fbfR1

Reply:
Yep, that change is not merged yet